octoprint_slic3r/profile.py

```diff
@@ -38,148 +38,170 @@ class SeamPositions(object):
+    gcode_comments = 0, 				# 0 (unchecked) or 1 (checked)
+    vibration_limit = 0,
+
+    filament_diameter = 3,
+    extrusion_multiplier = 1,
+    temperature = 200,
+    first_layer_temperature = 200,
+    bed_temperature = 0,
+
+    travel_speed = 130,
+    perimeter_speed = 30,
+    small_perimeter_speed = 30,
+    external_perimeter_speed = "70%",
+    infill_speed = 60,
+    solid_infill_speed = 60,
+    top_solid_infill_speed = 50,
+    support_material_speed = 60,
+    support_material_interface_speed = "100%",
+    bridge_speed = 60,
+    gap_fill_speed = 20,
+    first_layer_speed = "30%",
+
+    perimeter_acceleration = 0,
+    infill_acceleration = 0,
+    bridge_acceleration = 0,
+    first_layer_acceleration = 0,
+    default_acceleration = 0,
+
+    layer_height = 0.3,
+    first_layer_height = 0.35,
+    infill_every_layers = 1,
+    solid_infill_every_layers = 0,
+
+    perimeters = 3,
+    top_solid_layers = 3,
+    bottom_solid_layers = 3,
+    fill_density = "40%",
+    fill_angle = 45,
+    fill_pattern = FillPatterns.HONEYCOMB,
+    start_gcode = None,
+    end_gcode = None,
+    layer_gcode = None,
+    toolchange_gcode = None,
+    seam_position = SeamPositions.ALIGNED,
+    external_perimeters_first = 0, 			# 0 (unchecked) or 1 (checked)
+    spiral_vase = 0, 					# 0 (unchecked) or 1 (checked)
+    only_retract_when_crossing_perimeters = 0,		# 0 (unchecked) or 1 (checked)
+    solid_infill_below_area = 70,
+    infill_only_where_needed = 0, 			# 0 (unchecked) or 1 (checked)
+    infill_first = 0, 					# 0 (unchecked) or 1 (checked)
+
+    extra_perimeters = 1, 				# 0 (unchecked) or 1 (checked)
+    avoid_crossing_perimeters = 0, 			# 0 (unchecked) or 1 (checked)
+    thin_walls = 1, 					# 0 (unchecked) or 1 (checked)
+    overhangs = 0,
+
+    support_material = 0, 				# 0 (unchecked) or 1 (checked)
+    support_material_threshold = 0,
+    support_material_pattern = SupportPatterns.HONEYCOMB,
+    support_material_spacing = 2.5,
+    support_material_angle = 0,
+    support_material_interface_layers = 3,
+    support_material_interface_spacing = 0,
+    raft_layers = 0,
+    support_material_enforce_layers = 0,
+    dont_support_bridges = 0, 				# 0 (unchecked) or 1 (checked)
+
+    retract_length = 1,
+    retract_speed = 30,
+    retract_restart_extra = 0,
+    retract_before_travel = 2,
+    retract_lift = 0,
+    retract_layer_change = 1,
+    wipe = 0,
+
+    retract_length_toolchange = 1,
+    retract_restart_extra_toolchange = 1,
+
+    cooling = 0, 					# 0 (unchecked) or 1 (checked)
+    min_fan_speed = "35",
+    max_fan_speed = "100",
+    bridge_fan_speed = "100",
+    fan_below_layer_time = 60,
+    slowdown_below_layer_time = 30,
+    min_print_speed = 10,
+    disable_fan_first_layers = 1,
+    fan_always_on = 0,					# 0 (unchecked) or 1 (checked)
+
+    skirts = 1,
+    skirt_distance = 6,
+    skirt_height = 1,
+    min_skirt_length = 0,
+    brim_width = 0,
+
+    duplicate_distance = 6,
```

Comment:
I think they should be gone. I may be wrong, but executing the script I mentioned those are not settings available in Slic3r profile, so seems to happen that they are used when the user selects the option 'more' or 'scale...'. The following is just my opinion, but I think that for this version it does not make sense to have them included if the user cannot see how the result would be (i.e. how the objects would be placed or the size of the scaled object or objects).